README.md

```diff
@@ -62,6 +62,15 @@ docker run -p 3000:3000 tdm:v1.0.0
 
 The frontend will be accessible at `http://localhost:3000/` and the swagger configuration at `http://localhost:3000/swagger-config`.
 
+## Development mode
+
+To use the dev mode add the `http://localhost:3000` prefix to the servers url in `tdm-be/src/swagger/swagger.js`.
+
+Then run `make update-front-api`.
+
+You also need a ngrok host for the external host in `tdm-be/config/devlopement.json`
```

Comment:
<!-- Typo comments are like todo:, where the main issue is a mispelling.  -->
**typo:** ‏
```suggestion
You also need a ngrok host for the external host in `tdm-be/config/development.json`
```

Reply:
like this ?
See commit below.